src/corewlan.cs

```diff
@@ -544,4 +544,34 @@ interface CWNetworkProfile : NSCoding, NSSecureCoding, NSCopying, NSMutableCopyi
 	interface CWMutableNetworkProfile : NSCoding, NSSecureCoding, NSCopying, NSMutableCopying  
 	{
 	}
+
+    [BaseType (typeof (NSObject))]
```

Comment:
It looks like your editor is configured to indent using spaces, we're using tabs instead.